.github/workflows/ci.yml

```diff
@@ -185,6 +185,30 @@ jobs:
           yarn add -s prettier
           yarn exec -s prettier -- --check src/**/*.{ts,tsx}
 
+
+  python-formatting:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
+
+      - name: Install dependencies
+        run: |
+          pip install black isort pyproject-flake8==5.0.4
```

Comment:
Put them in `requirements-dev.txt` file, like it usually happens for python code. And `requirements.txt` for script dependencies if needed.